exc9.rb

```diff
@@ -0,0 +1,11 @@
+lunch_order = {
+  "Ryan" => "wonton soup",
```

Comment:
Šī ir vecā hash sintakse.
Stilīgi ir rakstīt šitā: 
```ruby
lunch_order = {
  Ryan: 'wonton soup',
  Eric: 'hamburder',
  ...
}
```

būsi ofisā, izstāstīšu par atšķirībām